src/views/exit-interview.php

```diff
@@ -0,0 +1,247 @@
+	}
+
+	.stellarwp-telemetry-starter ul.questions li input[type="radio"] {
+		margin-top: 1px;
+		margin-right: 1rem;
+	}
+
+	.stellarwp-telemetry-starter ul.questions li label {
+		width: 100%;
+	}
+
+	.stellarwp-telemetry-starter ul.questions li textarea {
+		width: 100%;
+		height: 55px;
+		border: 1px solid #DFDFDF;
+		border-radius: 4px;
+		padding: 0.5rem;
+		margin-top: 0.5rem;
+		display: none;
+	}
+
+	.stellarwp-telemetry-starter ul.questions li.active textarea {
+		display: block;
+	}
+
+	.stellarwp-telemetry-starter .btn-primary,
+	.stellarwp-telemetry-starter .btn-grey {
+		padding: 8px 12px;
+		background: #0047FF;
+		border-radius: 4px;
+		color: #fff;
+		text-decoration: none;
+		border: 0;
+		outline: none;
+		transition: all 0.1s ease-in-out;
+		cursor: pointer;
+	}
+
+	.stellarwp-telemetry-starter .btn-primary:hover {
+		background: #0032b7;
+	}
+
+	.stellarwp-telemetry-starter .btn-grey {
+		background: #eee;
+		color: #000;
+	}
+
+	.stellarwp-telemetry-starter .btn-grey:hover {
+		background: #ddd;
+	}
+</style>
+
+<script>
+	(function ($) {
+		let $body = $('body');
+		let redirectLink;
+
+		function showModal() {
+			$('.stellarwp-telemetry-starter.wrapper').addClass('active');
+		}
+
+		function closeModal( $this ) {
+			$this.closest('.stellarwp-telemetry-starter.wrapper').removeClass('active');
+		}
+
+		function submit() {
+			let $form = $('.stellarwp-telemetry-starter.wrapper').find('form');
+
+			$.ajax({
+				url: ajaxurl,
+				type: 'POST',
+				data: {
+					action: '<?php echo Exit_Interview_Subscriber::AJAX_ACTION; ?>',
+					nonce: '<?php echo wp_create_nonce( Exit_Interview_Subscriber::AJAX_ACTION ); ?>',
+					uninstall_reason: $form.find('input[name="uninstall_reason"]:checked').val(),
+				  	comment: $form.find('textarea').val(),
+				}
+			}).done(function (response) {
+				// Redirect to the plugin page.
+				window.location.href = redirectLink;
+			});
+		}
+
+		$body.on( 'click', '#the-list .deactivate > a', function ( e ) {
+			if ( 0 === $( this ).next( '[data-module-id].telemetry-plugin-id' ).length ) {
+				return true;
+			}
+
+			e.preventDefault();
+			redirectLink = $(this).attr('href');
+			showModal();
+		});
+
+		$body.on( 'click', '[data-js="close-modal"]', function ( e ) {
+			e.preventDefault();
+			closeModal( $(this) );
+		});
+
+		$body.on( 'click', '[data-js="submit-telemetry-starter"]', function ( e ) {
+			e.preventDefault();
+			submit( $(this) );
+		});
+
+		$body.on( 'change', '[name="uninstall_reason"]', function () {
+			let $this = $(this);
+			let $wrapper = $this.closest('li');
+			let $reason = $wrapper.find('[name="comment"]');
+
+			if ( ! $reason.length ) {
+				return;
+			}
+
+			$body.find('ul.questions li.active').removeClass('active');
+			$body.find('ul.questions li [name="comment"]').val('');
+			$wrapper.addClass('active');
+		});
+	})(jQuery);
+</script>
+
+<div class="stellarwp-telemetry-starter wrapper">
+	<section class="stellarwp-telemetry-starter modal">
```

Comment:
I want to avoid using `starter` anywhere in the library. `stellarwp-telemetry` should be enough here.